framework/src/meshgenerators/SideSetsFromBoundingBoxGenerator.C

```diff
@@ -57,8 +66,12 @@ SideSetsFromBoundingBoxGenerator::SideSetsFromBoundingBoxGenerator(
     _input(getMesh("input")),
     _location(parameters.get<MooseEnum>("location")),
     _block_id(parameters.get<SubdomainID>("block_id")),
-    _boundary_id_old(parameters.get<std::vector<BoundaryName>>("boundary_id_old")),
-    _boundary_id_new(parameters.get<boundary_id_type>("boundary_id_new")),
+    _boundaries_old(parameters.isParamSetByUser("boundary_id_old")
+                        ? parameters.get<std::vector<BoundaryName>>("boundary_id_old")
+                        : parameters.get<std::vector<BoundaryName>>("boundaries_old")),
```

Comment:
this will error weirdly
you should have a default value for boundaries_old (for now, it s temporary)

then in the body of the constructor, check that one (at least and at most) of these two parameters is passed